docs/internet-identity-spec.adoc

```diff
@@ -536,6 +540,23 @@ This flow is the boring default
 +
 (This could include a button to go to the management screen, or maybe this _is_ the management screen with a info box.)
 
+=== Flow: Setup recovery
+1. The user is offered two options for recovery
+  * A security key
+  * A BIP-39 seed phrase
+  * (Skip?)
```

Comment:
Well, let’s decide.

I’m in favor of Skip. If only so that we can test stuff more easily :-) But also to be less patronizing.

Reply:
I did add a skip button, so yeah I'll update this one.